documentation/modules/con-standard-address-space.adoc

```diff
@@ -4,11 +4,11 @@
 
 [id='con-standard-address-space-{context}']
 = Standard Address Space
-The standard address space is the default address space in {ProductName}. It consists of an AMQP router network in combination with attachable storage units. The implementation of a storage unit is hidden from the client and the routers with a well-defined API. This address space type is appropriate when you have many connections and addresses. However, it has the following limitations: 
+The standard address space is the default address space in {ProductName}. It consists of an AMQP router network in combination with attachable storage units. The implementation of a storage unit is hidden from the client and the routers with a well-defined API. This address space type is appropriate when you have many connections and addresses. However, the standard address space has the following limitations:
 
 * No transaction support
 * No message ordering
-* No selectors on queues
+* No browsing or selectors on queues
```

Comment:
I'd possibly leave these as two separate items ('no selectors on queues' ''no browsing of queues '). Not a big issue though if you prefer this form.

Reply:
Nope, that was a typo on my part. Will fix and keep them as two separate items.